ci/build_wheel_librapidsmpf.sh

```diff
@@ -40,3 +40,6 @@ python -m auditwheel repair \
     ${package_dir}/dist/*
 
 ./ci/validate_wheel.sh "${package_dir}" "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"
+
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")"
+RAPIDS_PY_WHEEL_NAME="librapidsmpf_${RAPIDS_PY_CUDA_SUFFIX}" rapids-upload-wheels-to-s3 cpp "${RAPIDS_WHEEL_BLD_OUTPUT_DIR}"
```

Comment:
These `rapids-upload-wheels-to-s3` calls can be reverted on 25.08, but need to be here on 25.06 because https://github.com/rapidsai/shared-workflows/pull/364 targeted `branch-25.08`.